### ISSUE TITLE
[Term Entry] Neural Networks ai gradient descent

### ISSUE THREAD

**Issue by codecademy-docs** [Codecademy/docs#4526]
### Reason/inspiration (optional)

"We would like a new term entry in the `AI` concept for [neural-network](https://www.codecademy.com/resources/docs/ai/neural-networks): gradient descent. The entry should go in a new file under `docs/content/ai/concepts/neural-networks/terms/gradient-descent/gradient-descent.md`.

The entry should describe the term and how it is used. Also include: 
- An "Example" section that explains to readers how to use concept for gradient descent.
- Should contain information for cncept and syntax.
- Should have a Python codebyte with a working code snippet showcasing this concept.
Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue."

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

**Comment by FuzzyNum** [comment]
I would like to be assigned to this entry. I have experience writing and training neural networks, and I have proficiency in gradient descent. Also, my primary language is Python. Let me know.

**Comment by mamtawardhani** [comment]
Hey @FuzzyNum!
You’re assigned 🎉 In addition to the documents linked in the description, please also look at the [Contribution Guide](https://www.codecademy.com/resources/docs/contribution-guide). After creating a PR, the maintainer(s) (with the collaborator label) will add comments/suggestions to address any revisions before approval.

Is this your first contribution to Codecademy Docs? If so, we’re curious to know how you found out about contributing to Docs.

**Comment by ishg-153** [comment]
@FuzzyNum It's been almost 2 months since you were assigned this issue. As there was no response from your end, I am unassigning you.

**Comment by JoseCarlosInchausti** [comment]
Hello! I am interested in getting assigned to this issue, but I was hoping to get some clarification if possible. What does syntax refer to in this context? Should gradient descent be explained in relation to a particular library (like Sklearn)? I have a lot of experience with ML, but I want to make sure I understand the prompt correctly. Thank you!

**Comment by ishg-153** [comment]
Hey @JoseCarlosInchausti, You're assigned!🎉 You can look through the [Contribution Guide](https://github.com/Codecademy/docs/blob/main/.github/CONTRIBUTING.md) to know more about contributing to Codecademy Docs. Check out the [Content Standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [Style Guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md), they’ll help you with creating your entry. 

I see this is your first time contributing to Codecademy Docs. How did you find out about us?

**Comment by ishg-153** [comment]
@JoseCarlosInchausti Since this term entry is under `ai` topic, this term entry is supposed to be theoretical. You can disregard the description. Just make sure that the entry doesn't get too long.

**Comment by Sriparno08** [comment]
Hey @JoseCarlosInchausti, it's been a month and there's no PR from your side. Are you still working on this issue?

Note: If you don't respond within two days, you'll be unassigned from this issue unfortunately.

**Comment by Sriparno08** [comment]
Hey @JoseCarlosInchausti, you're unassigned due to inactivity.

**Comment by deenovita** [comment]
Hello, may I work on this issue? thank you

**Comment by Sriparno08** [comment]
Hey @deenovita, you're assigned!

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md) and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

**Comment by Sriparno08** [comment]
Hey @deenovita, it's been a month and there's no PR from your side. Are you still working on this issue?

Note: If there's no response from your side in 2 days, you will be unassigned unfortunately.

**Comment by deenovita** [comment]
Hello @Sriparno08, I'm still working on this issue. Sorry for taking too long to get back to you

**Comment by Sriparno08** [comment]
No worries, @deenovita. We are all waiting for your PR!

**Comment by mamtawardhani** [comment]
> Hello @Sriparno08, I'm still working on this issue. Sorry for taking too long to get back to you

Hey @deenovita are you still working on this?

**Comment by mamtawardhani** [comment]
Unassigning as author is inactive
